### ISSUE TITLE
[BUG] SKCanvasView does not appear in accessibility data

### ISSUE THREAD

**Issue by rachelmoor** [mono/SkiaSharp#1008]
**Description**

The SKCanvasView does not appear in the accessibility data on my Xamarin Forms UWP application. I used SKCanvasView as the base class in a custom control, and the SVG I want to display appears, but I cannot see this node using Inspect.

**Code**
[SkiaSharpBug.zip](https://github.com/mono/SkiaSharp/files/3838377/SkiaSharpBug.zip)

Note: the SVGImage class is from [Nick Peppers github repo](https://github.com/nickpeppers/SkiaSharpSvgImage/tree/master/SkiaSharpSvgImage)

**Expected Behavior**

The SVG image is visible in the accessibility data.

**Actual Behavior**

The SVG image is not visible in the accessibility data (see screenshot below)

**Basic Information**

- Version with issue: 1.68.1-rc.165
- Last known good version:  N/A
- IDE:  VS 2019
- Platform Target Frameworks:
  - UWP:  17134
  - Windows Classic:  Windows 10 v 1803 Build 17134

- Target Devices:   desktop computer

**Screenshots**

This is a screenshot of the UWP app attached. As you can see, the first text is visible in the accessibility data and the bottom text, but the SVG image did not appear.

![image](https://user-images.githubusercontent.com/13911774/68713980-e031b580-0553-11ea-91cd-2224f6dd3f98.png)

**Reproduction Link**

I attached a zip file containing my sample application.
